sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -62,6 +62,8 @@ namespace Azure { namespace Core { namespace Http {
      * @remark More about this option:
      * https://curl.se/libcurl/c/CURLOPT_CAINFO_BLOB.html
      *
+     * @warning Requires libcurl >= 7.44.0
+     *
      */
     std::string PemEncodedExpectedRootCertificates;
```

Comment:
Because we are not if-defing this out for older versions, any customer who sets this with older version of curl will be surprised to see it ignored.

Throwing is a good signal that the customer has to fix something, rather than silently ignoring the setting.